### DISCUSSION TITLE
Main Menu - Icon and Halo anomoly

### DISCUSSION THREAD

**Discussion by KBKreisler** [Jyers/Marlin#1452] [Q&A] [answered]
Hello - 
The issue I am running into demonstrated in video.

https://user-images.githubusercontent.com/89944718/135559246-9717f903-b02e-4359-ae6d-3efe55802f17.mp4

MB is SKR 2B - DWIN from ender 3 V2
working on compiling firmware for a MB swap.

All the menus appear to work, fine and function seems to be there - just get residual halo and text instead of icons.
This happens in bugfix when using any of the DWIN options - does not occur when loading stable release.
Wondering if anyone has seen anything similar to this?

**Comment by adam3654** [comment]
I don't have an answer as to why, but I just tried flashing the upstream bugfix branch of marlin and I had the exact same thing happen to me. I am running a stock 4.2.2 motherboard though. I tried a filament change and it locked up after clicking ok after inserting the new filament. Back to Jyers I went, and all is back to normal.

**Comment by Jyers** [comment, marked as answer]
The actual issue with this is that upstream marlin assumes we would use a 7.ico file, when the default is a 9.ico. The latest source takes this into account. Compile and enjoy your icons :)
